en/sensor/rangefinders.md

```diff
@@ -68,31 +68,43 @@ PX4 also supports the Bebop rangefinder.
 - [EKF2\_RNG\_A\_\*](../advanced_config/parameter_reference.md#EKF2_RNG_AID) - limits and consistency checks when using the sensor for the [range aid feature](../advanced_config/parameter_reference.md#feature).
 
 
 ### Usage/Features {#features}
 
-Rangefinders can be enabled to support flight in two ways:
-1. Set [EKF2_HGT_MODE](../advanced_config/parameter_reference.md#EKF2_HGT_MODE) to *Range finder* (`2`). 
-   This makes the rangefinder the primary source of height estimation (the default altitude sensor is the barometer).
-1. Set [EKF2_RNG_AID](../advanced_config/parameter_reference.md#EKF2_RNG_AID) to `1`. 
-   This makes the vehicle use the rangefinder as the primary source when it is safe to use, but will otherwise use the sensor specified in `EKF2_HGT_MODE`.
-   * Specifically, the rangefinder is enabled when:
-     * velocity < [EKF2_RNG_A_VMAX](../advanced_config/parameter_reference.md#EKF2_RNG_A_VMAX)
-     * distance to ground < [EKF2_RNG_A_HMAX](../advanced_config/parameter_reference.md#EKF2_RNG_A_HMAX)
-   * Other parameters affecting "Range aid" are prefixed with `EKF2\_RNG\_A\_`.
+Distance sensors can be enabled to support flight in several ways:
+1. Enable the *Range sensor* as the primary source of height estimation by setting [EKF2_HGT_MODE=2](../advanced_config/parameter_reference.md#EKF2_HGT_MODE) (the default altitude sensor is the barometer). 
+   - This setting should only be used when for operation over a flat surface as the local NED origin will move up and down with ground level.
+   
+1. Enable *Range Aid* by setting [EKF2_RNG_AID=1](../advanced_config/parameter_reference.md#EKF2_RNG_AID). 
+   - This makes the vehicle use the rangefinder as the primary source of height estimation *when it is safe to use* (during low speed/low altitude operation), but will otherwise use the sensor specified in `EKF2_HGT_MODE`.
+   - It is primarily intended for takeoff and landing, where baro interference from rotor wash is excessive and can corrupt EKF state estimates. 
+   - Specifically, the rangefinder is enabled when:
+     - velocity < [EKF2_RNG_A_VMAX](../advanced_config/parameter_reference.md#EKF2_RNG_A_VMAX)
+     - distance to ground < [EKF2_RNG_A_HMAX](../advanced_config/parameter_reference.md#EKF2_RNG_A_HMAX)
+   - Other parameters affecting "Range aid" are prefixed with `EKF2_RNG_A_`.
+
+1. Enable rangefinder-supported *terrain following* by setting [MPC_ALT_MODE=1](../advanced_config/parameter_reference.md#MPC_ALT_MODE).
+   - In this mode the vehicle altitude moves up and down with terrain height variation.
+   - At altitudes above the distance sensor effective range the vehicle reverts to using height above the home position (as provided by the primary altitude source defined in `EKF2_HGT_MODE`).
+
+1. Enable rangefinder-supported *terrain holding* by setting [MPC_ALT_MODE=2](../advanced_config/parameter_reference.md#MPC_ALT_MODE).

```

Comment:
Hi @bkueng,
Just want to confirm I get terrain following and hold correctly. 

My assumption for terrain following is that if I were in any manual mode where sticks control position, yaw, altitude, and move forward (while range finder able to act) I would follow terrain at that altitude. 

For terrain hold, I think this might only apply when in an altitude hold or position mode. But basically the altitude is locked based on terrain height from sensor rather than baro. Correct?

Reply:
@bkueng Not clear to me either - who would know?

EKF is only used on copters/VTOL right? (ie I was thinking maybe the RNG_AID might be useful on platforms other than copter, and this could be the essential difference?)

What is clear is that the docs on EKF2_RNG_AID will need an update because they attempt and fail to answer this question (ie they indicate that MPC_ALT_MODE  should be used instead unless there are baro issues, but in fact the description of behaviour is the same).

>
I guess setting EKF2_RNG_AID=1